litmus-portal/frontend/src/models/redux/workflow.ts

```diff
@@ -11,6 +11,9 @@ export interface WorkflowData {
   weights: experimentMap[];
   isCustomWorkflow: boolean;
   clusterid: string;
+  details: string;
```

Comment:
I don't think you need these in redux, we are not using these details,We are just showing them on frontend.
You can do that without redux.